test/unit/math/prim/mat/meta/VectorViewMvt_test.cpp

```diff
@@ -0,0 +1,51 @@
+#include <stan/math/prim/mat.hpp>
+#include <gtest/gtest.h>
+#include <iostream>
+#include <stdexcept>
+#include <vector>
+
+// Just test the use case where a single matrix is passed in to VectorViewMvt.
+// The case where a std::vector of matrices is passed in is broken and will be refactored.
```

Comment:
The language is complicated around Eigen.  You are actually passing in vectors, not matrices, though it's confusing because Eigen's vector type is `Matrix` with `1` for template parameter for number of rows.  So how general is `VectorViewMvt` supposed to be?